offline-deployment-using-ansible.md

```diff
@@ -6,6 +6,12 @@ aliases: ['/docs/dev/offline-deployment-using-ansible/','/docs/dev/how-to/deploy
 
 # Deploy TiDB Offline Using TiDB Ansible
 
+> **Warning:**
+>
+> For production environments, it is recommended that you [deploy TiDB using TiUP offline](/production-offline-deployment-using-tiup.md). Since v4.0, PingCAP no longer provides support for deploying TiDB using TiDB Ansible (deprecated). If you really need to use it for deployment, be aware of any risk. You can [import the TiDB cluster deployed by TiDB Ansible to TiUP](/upgrade-tidb-using-tiup.md#import-the-tidb-ansible-cluster-to-tiup).
+>
+> If you only want to try out TiDB and explore new features, refer to [Quick Start Guide](/quick-start-with-tidb.md) or [deploy TiDB using Docker Compose](/deploy-test-cluster-using-docker-compose.md) on a single machine.
```

Comment:
Since the Docker Compose deployment document is no longer maintained, I suggest removing it here.
```suggestion
> If you only want to try out TiDB and explore new features, refer to [Quick Start Guide for the TiDB Database Platform](/quick-start-with-tidb.md).
```